### ISSUE TITLE
Focus Next/Prev Pane command won't focus console-only panel

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7880]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.869
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- Close all the optional tabs in the Console panel (e.g. Terminal, Jobs)
- Use Focus Next Pane (F6) or Focus Previous Pane (Shift+F6) to move focus between panes
- Watch what happens when you encounter the console

### Describe the problem in detail
- Focus will not enter the Console from either direction

### Describe the behavior you expected
- Should be able to navigate to Console even when it's the only pane in the console panel

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kfeinauer** [comment]
Verified fixed 1.4.1069
